backend/app/main.py

```
@@ -328,66 +331,15 @@ def _rebuild_with_timeout():
-        "errored": len(errored),
-        "queued_items": queued,
-    }
+    # Periodic rate limiter cleanup to prevent memory leaks
+    async def _rate_limit_cleanup_loop():
+        from app.middleware.rate_limit import get_limiter
+        limiter = get_limiter()
+        while True:
+            await asyncio.sleep(300)  # every 5 minutes
+            await limiter.cleanup()
 
+    asyncio.create_task(_rate_limit_cleanup_loop())
```

Ruff (0.15.6)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)